fronts-client/src/components/feed/RecipeSearchContainer.tsx

```diff
@@ -38,34 +39,53 @@ const FeastSearchContainerComponent = ({
   rightHandContainer,
   recipes,
   chefs,
-}: Props) => (
-  <React.Fragment>
-    <InputContainer>
-      <TextInputContainer>
-        <TextInput placeholder="Search recipes" displaySearchIcon />
-      </TextInputContainer>
-      <ClipboardHeader />
-    </InputContainer>
-    <FixedContentContainer>
-      <SearchResultsHeadingContainer>
-        <SearchTitle>
-          {'Results'}
-          <ShortVerticalPinline />
-        </SearchTitle>
-      </SearchResultsHeadingContainer>
-      {Object.values(recipes).map((recipe) => (
-        <RecipeFeedItem key={recipe.id} recipe={recipe} />
-      ))}
-      {Object.values(chefs).map((chef) => (
-        <ChefFeedItem
-          key={chef.id}
-          chef={chef}
-        /> /*TODO: as of now, added rendering of chefs just after the recipes. Need
-      to change when "search-chefs" action gets finalised*/
-      ))}
-    </FixedContentContainer>
-  </React.Fragment>
-);
+}: Props) => {
+  const [selectedOption, setSelectedOption] = useState<string>('recipeFeed');
```

Comment:
Tiny point - I've found that if you specify a default value like this, Typescript can infer the type and you don't have to specify it directly with `<string>`

Reply:
My preference for this is to use numbers, defined as enums (https://www.typescriptlang.org/docs/handbook/enums.html); reason is that it's quicker for the machine to compare and process numbers than strings.  However these days I guess that the performance difference is negligible though I would encourage the use of enums or constants over "magic value" strings because if you put a typo in an enum value the compiler complains wheras if you put a typo in a "magic value" string the app breaks

Reply:
Makes more sense. updated with enum. thanks @fredex42